duet/views/google_uac_android_activation.view.lkml

```diff
@@ -0,0 +1,123 @@
+view: google_uac_android_activation {
+
+  derived_table: {
+    sql: WITH uac AS (
+         SELECT
+           date,
+           REGEXP_REPLACE(campaign_name, '(.*) - NULL', '\\1') as campaign,
+           SUM(spend) as cost
+         FROM
+           mozdata.google_ads.daily_campaign_stats
+         WHERE
+            campaign_name NOT LIKE '%iOS%'
+            AND date >= '2022-12-01'
+         GROUP BY
+            1, 2
+        ), activation AS (
+         SELECT
+           first_seen_date as date,
+           REGEXP_REPLACE(adjust_campaign, '([\\w]+).*', '\\1') as campaign,
+           SUM(activated) as activated,
+           COUNT(*) as new_profiles,
+         FROM `moz-fx-data-shared-prod.fenix.new_profile_activation`
+         WHERE submission_date >= '2022-12-01'
+           AND first_seen_date >= '2022-12-01'
+           AND adjust_network = 'Google Ads ACI'
+           AND adjust_campaign LIKE 'Mozilla_%'
+         GROUP BY 1, 2
+        )
+        SELECT
+          date,
+          campaign,
+          CASE WHEN campaign LIKE '%_US_%' OR campaign LIKE '%_CA_%' OR campaign LIKE '%NA%' THEN 'NA' ELSE 'EU' END AS region,
+          REGEXP_EXTRACT(campaign, '.*(Event\\d).*') as event,
+          REGEXP_EXTRACT(campaign, '.*(Group\\d).*') as group_number,
+          REGEXP_EXTRACT(campaign, 'Mozilla_FF_UAC_[\\w]{2}_([\\w]{2})_[\\w]{2}_Group\\d_Event\\d') as country_code,
```

Comment:
I understand that this is only available on the newest campaigns, but it might not be a good idea to include it here - there's too many caveats. If, for example, someone filters for `date >= "2023-01-01" AND country_code = "DE"`, they will be filtering _out_ DE data from 2023-01-1 through 2023-05-01 (thereabouts), though of course that DE data would be combined w/ other countries

Reply:
What if I rename to "campaign country code"? I did explicitly include it in the description of the field in the explore, but I understand there is a risk people will miss it.
This field is important because it was something Sally was using in our working session. I think it's less risky than people trying to guess by for example creating a filter with campaign name containing "DE" without realizing it's a language for campaigns in Switzerland for example.

Reply:
Hm, good point. It's a small user base so we can try to make sure it's being used correctly. `campaign_country_code` seems 👍